.env.example

```
@@ -0,0 +1,67 @@
+# TESTING YOUR SETUP:
+# - Run your development server
+# - Submit a test form on your website
+# - Check your email for the notification
+# - Verify submissions appear in your Formspree dashboard
+#
+# TROUBLESHOOTING:
+# - Make sure the form ID is copied exactly (case-sensitive)
+# - Check that your .env file is in the project root
+# - Restart your development server after changing .env
+# - If you exceed 50 submissions/month, upgrade to a paid plan
+# =============================================================================
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)